src/main/java/ru/mambetpages/dto/GetArticleFromProfileDto.java

```diff
@@ -0,0 +1,13 @@
+package ru.mambetpages.dto;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.Data;
+
+@Data
+public class GetArticleFromProfileDto {
+    @Schema(description = "Общее количество страниц")
+    private int pagesCount = 10;
```

Comment:
Do not preset pagesCount to 10, just use private Integer pagesCount;
And dont forget to set this value in mock controller